source/EULA-Non-Windows.txt

```diff
@@ -0,0 +1,41 @@
+MICROSOFT SOFTWARE LICENSE TERMS
```

Comment:
Do your team have a guideline on which license to include?
You may be fine, but it would be clear following the guideline since this project has no dependency on Windows.

Reply:
removed the Windows one as you mentioned not necessarily due to podspec requirements. I will check with the team the right licences to add.